source/native/symbulation_sgp.cc

```diff
@@ -36,8 +36,11 @@ int symbulation_main(int argc, char * argv[])
 
   config.Write(std::cout);
   emp::Random random(config.SEED());
-
-  SGPWorld world(random, &config, DefaultTasks);
+  TaskSet task_set = LogicTasks;
+  if (config.TASK_TYPE() == 0){
+     task_set = SquareTasks;
+  }
```

Comment:
Make sure to set the DefaultTasks for the other flag value so you don't mess up other people's code